xml/cl.xml

```diff
@@ -1357,8 +1357,9 @@ server's OpenCL/api-docs repository.
 
     <enums name="cl_command_buffer_state_khr" vendor="Khronos">
         <enum value="0"            name="CL_COMMAND_BUFFER_STATE_RECORDING_KHR"/>
-        <enum value="1"            name="CL_COMMAND_BUFFER_STATE_EXECUTABLE_KHR"/>
-        <enum value="2"            name="CL_COMMAND_BUFFER_STATE_PENDING_KHR"/>
+        <enum value="1"            name="CL_COMMAND_BUFFER_STATE_FINALIZED_KHR"/>
```

Comment:
We should add CL_COMMAND_BUFFER_STATE_FINALIZED_KHR to the  `New Enums` section of either cl_khr_command_buffer or cl_khr_command_buffer_mutable_dispatch
* https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#_new_enums_39
* https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#_new_enums_41